rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewGroupHierarchyChildViewAddEvent.java

```diff
@@ -0,0 +1,44 @@
+package com.jakewharton.rxbinding.view;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.view.View;
+import android.view.ViewGroup;
+
+/**
+ * A child view add event on a viewgroup.
```

Comment:
This Javadoc does not match the other events which include a warning. Also "viewgroup" is not a word and should probably be a `{@link ..}`